.storybook/main.js

```diff
@@ -1,44 +1,25 @@
 const fs = require('fs');
 const path = require('path');
+const globby = require('globby');
 
 module.exports = {
-  stories: [
-    '../{packages,packages-node}/*/README.md',
-    '../{packages,packages-node}/*/docs/*.md',
-    '../{packages,packages-node}/*/docs/!(assets)**/*.md',
-    '../packages/helpers/*/README.md',
-    '../docs/README.md',
-    '../docs/**/*.md',
-    '../README.md',
-    '../demo/README.md',
-    '../demo/docs/*.md',
-  ],
-  addons: [
-    // order of tabs in addons panel
-    'storybook-prebuilt/addon-actions/register.js',
-    'storybook-prebuilt/addon-knobs/register.js',
-    'storybook-prebuilt/addon-a11y/register.js',
-    'storybook-prebuilt/addon-docs/register.js',
-    // no tab in addons panel (e.g. load order does not matter here)
-    'storybook-prebuilt/addon-backgrounds/register.js',
-    'storybook-prebuilt/addon-links/register.js',
-    'storybook-prebuilt/addon-viewport/register.js',
-  ],
-  addons: [
-    'storybook-prebuilt/addon-docs/register.js',
-    'storybook-prebuilt/addon-actions/register.js',
-    'storybook-prebuilt/addon-knobs/register.js',
-    'storybook-prebuilt/addon-a11y/register.js',
-    'storybook-prebuilt/addon-backgrounds/register.js',
-    'storybook-prebuilt/addon-links/register.js',
-    'storybook-prebuilt/addon-viewport/register.js',
-  ],
-  esDevServer: {
-    nodeResolve: true,
-    watch: true,
-    open: true,
-  },
-  rollup: config => {
+  stories: globby
+    .sync([
+      './{packages,packages-node}/*/README.md',
+      './{packages,packages-node}/*/docs/*.md',
+      './{packages,packages-node}/*/docs/!(assets)**/*.md',
+      './{packages,packages-node}/*/docs/validate/!(assets)**/*.md',
+      './packages/helpers/*/README.md',
+      './docs/README.md',
+      './docs/**/*.md',
+      './README.md',
+      './demo/README.md',
+      './demo/docs/*.md',
+    ])
```

Comment:
With globby like this we can at least use JS to filter out stuff that the glob caught that we don't want, makes it a bit easier than puzzling for multiple days on a glob pattern that works 😓